src/main/docker/docker-compose.yml

```diff
@@ -162,36 +353,41 @@ services:
       - spring.mail.password=
       - spring.mail.properties.mail.smtp.auth=false
       - spring.mail.properties.mail.smtp.starttls.enable=false
-      - er.url=eiffel-er:8082
+      - er.url=http://${HOST}:8084/search/
```

Comment:
It hink you should use internal hostname on each service instead of HOST ip address.
By using HOST IP address the connection will exit docker host and enter the Docker host again. This is an issue if you are working from home and connected with VPN, the VPN tunnel will eat up the connection ER connection when it exits the dockerhost and it will never find the ER service. You will probably get a connection timeout or something in that situations. So better to use the Eiffel-ER internal Docker hostname / alias, e.g.: "eiffel-er" .

Reply:
Fixed.